io_section.tex

```diff
@@ -0,0 +1,257 @@
+out to 384 nodes (approx. 14K cores). A description of the work plus full results can be found in \cite{Adams2018}. 
+
+For the 2019 report, the focus is on two main areas: strong scaling when outputting regular
+diagnostics and also tests of in-situ post-processing. In particular tests have been done of time-meaning and also regridding 
+from the LFRic native cubed-sphere mesh to a regular lat-lon mesh. Both of these are use cases from the upcoming Aquaplanet
+runs and also evaluation of XIOS regridding is of interest to the wider Next Generation Modelling Systems programme as it
+could inform how downstream processing will be handled. For instance whether LFRic can deliver Level 1 data directly to StaGE.
+It could also make verification of LFRic easier if the data can be directly produced on a lat-lon mesh. The aim of the post-processing
+experiments described here is primarily to get an idea of computational overhead in terms of runtime and memory, 
+so there is no in depth analysis of scientific accuracy apart from verifying that the jobs have produced the right kind of output. 
+
+\subsection{Job Setup}
+All tests were performed on XCS. The executables were built using the LFRic Intel Fortran environment (17.0.0.098) 
+including the latest PSyclone 1.7.0 and XIOS at r1537 of the XIOS trunk. Build flags were the default fast-debug.
+
+The science configuration was the standard baroclinic wave test with timestep length dependent on mesh resolution. 
+For the purposes of the tests here: 900s for a C96 mesh and 180s for a C576 mesh.
+10 fields are currently output as diagnostics in this configuration which results in approximately 5 Gb data per
+output
+
+To keep within queue time limit the basic run is for 120 timesteps, with diagnostic frequency set to 20 timesteps.
+This makes for 6 writes over the course of the run.
+
+\subsection{Glossary of terms}
+
+ 
+\begin{itemize}
+  \item Client Ratio (CR).The percentage of time a client (model) task waits for the server. Ideally it should be as close to zero as possible.
+  \item Client I/O time. The actual time (seconds) that the client task spends in I/O related activity
+\end{itemize} 
+
+
+
+\subsection{Set 1: Strong scaling}
+
+This set of tests used a C576 mesh (approx 17Km horizontal resolution). Pure MPI only (\$OMP\_NUM\_THREADS=1). 
+I/O setup used 12 XIOS servers with lustre stripe set count to be the same. This was the best performing setup from
+the last set of I/O performance tests, so the assumption is that it is a good starting point.
+The basic run is for 120 timesteps, with diagnostic\_frequency set to 20 timesteps. This makes for 6 writes over the course of the run.
+
+\begin{table}[ht!]
+\scriptsize
+  \begin{center}
+    \caption{Strong Scaling: Runtime}
+    \label{tab:table1}
+     \begin{tabular}{|c|c|c|c|c|c|}
+      \textbf{Nodes} & \textbf{Max CR~(\%)} & \textbf{Max Client IO~(s)} & \textbf{Runtime~(s)} & \textbf{Runtime no IO~(s)} & \textbf{\% incr IO}\\
```

Comment:
Is it worth swapping the order of "Runtime~(s)" and "Runtime no IO~(s)" so that it reads as
<vanilla> || <what we're interested in here> || <% change >

Reply:
yep - that makes sense!